features/groups.feature

```diff
@@ -649,4 +649,22 @@ Feature: Groups
                         }
                     ]
                 }
-            """
\ No newline at end of file
+            """
+
+#   Delete deleteGroup scenarios
+#   successful return
+    Scenario: GET /v1/groups/{id} and checking the response status 200
+        Given group "test-group" exists in the database
+        When I GET "/v1/groups/test-group"
+        Then the HTTP status code should be "200"
+
+#   404 return
+    Scenario: GET /v1/groups/{id} and checking the response status 404
+        Given group "get-group-not-found" exists in the database
```

Comment:
It's a bit confusing that the given clause suggests the group `exists in the database`, but then when the get request is made it returns a 404? Consider if the `Given` line can be removed.